### DISCUSSION TITLE
Authoring tool accessibility support

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#6] [Specific, miscellaneous]
Content authoring tools (e.g., content management systems and no code tools) could do much more to support accessibility. *(more info in [Essential Components of Web Accessibility](https://www.w3.org/WAI/fundamentals/components/))*

W3C has a standard and supporting material:

* Authoring Tool Accessibility Guidelines [ATAG](https://www.w3.org/WAI/standards-guidelines/atag/), [Implementing ATAG](https://www.w3.org/TR/IMPLEMENTING-ATAG20/), and [ATAG at a Glance](https://www.w3.org/WAI/standards-guidelines/atag/glance/)
* [ATAG Report Tool](https://www.w3.org/WAI/atag/report-tool/), [Authoring Tools List of accessibility support](https://www.w3.org/WAI/tools-list/authoring/)
* outreach pages: [Standards to Make Your LMS Accessible](https://www.w3.org/WAI/standards-guidelines/atag/education/), [Make Your Social Media Platform Accessible](https://www.w3.org/WAI/standards-guidelines/atag/social-media/),  [Make Your No-Code Website Tool Accessible](https://www.w3.org/WAI/standards-guidelines/atag/no-code/), 

Do we want to prioritize work encouraging improvements in authoring tools?
Or, is this not a priority for W3C accessibility work for 2024?

**Comment by mitchellevan** [comment]
Two reasons for a strong up-vote from me.

1. Large need. So much of the web depends on non-developers to make it accessible. Making it possible to author accessible content is not enough; making it easier is vital.
2. Small investment. ATAG 2 might need only an update from WCAG 2.0 to 2.2 by reference, so it can be relevant for procurement regulations such as Europe's which use at least 2.1.
